### ISSUE TITLE
manually enable/disable entries

### ISSUE THREAD

**Issue by tomasabril** [modzero/fix-windows-privacy#1]
manually enable/disable entries instead of disabling all of them.
I wish to keep a few settings still enabled.

**Comment by devio** [comment]
I totally agree with you. I'm working on this feature for the next release, please stay tuned. Currently the only way to manually remove or add rules is to edit the XML files in c:\Program Files\modzero\fix-windows-privacy.

**Comment by revident** [comment]
@devio This issue is a blocker for me as well. These privacy changed fall into generally three categories.
* Want
* Do Not Want
* Do Not Know

This second category is critical, as there is never a one size fits all solution.
The third category is the most troublesome, and deserves it's own issue.
